### DISCUSSION TITLE
Png с прозрачностью в webp без черных полей

### DISCUSSION THREAD

**Discussion by mrsky1001** [Yuriy-Svetlov/compress-images#68] [Q&A]
При попытке преобразования png с прозрачностью в webp появляются черные поля вместо прозрачности. Вопрос - можно ли задать цвет для замены прозрачности?

**Comment by Yuriy-Svetlov** [comment]
> При попытке преобразования png с прозрачностью в webp появляются черные поля вместо прозрачности. Вопрос - можно ли задать цвет для замены прозрачности?

Здравствуйте.

Вам нужно обратиться к документации движка [webp](https://developers.google.com/speed/webp/) .

Дополнительная информация:
[docs/cwebp](https://developers.google.com/speed/webp/docs/cwebp)

> -lossless
> Encode the image without any loss. For images with fully transparent area, the invisible pixel values (R/G/B or Y/U/V) will be preserved > only if the -exact option is used.

> -alpha_q int
> Specify the compression factor for alpha compression between 0 and 100. Lossless compression of alpha is achieved using a value of > > 100, while the lower values result in a lossy compression. The default is 100.
